azure-pipelines.yml

```diff
@@ -115,7 +115,7 @@ jobs:
       displayName: 'Install dependencies (i386 Linux)'
```

Comment:
> > i can't see the "re-run jobs" option mentioned in contributing.html to restart that crashed check

see https://nim-lang.github.io/Nim/contributing.html#the-cmdgit-stuff-continuous-integration-ci

> follow these instructions to only restart the jobs that failed:

(feel free to improve wording if unclear; note that some of these instructions may require sufficient permisssions in nim repo, or not, i fogot)

Reply:
They only work if you have commit permission in nim-lang/nim afaik.

Reply:
PR welcome to clarify this in the doc (and whether it's for both azure and github actions or just 1 of those)